boxoffice/views/admin_report.py

```diff
@@ -49,19 +47,74 @@ def row_handler(row):
     (ItemCollection, {'id': 'ic_id'}, 'item_collection'),
     permission='org_admin')
 def attendees_report(item_collection):
-    headers = ['receipt_no', 'ticket_no', 'ticket_id', 'ticket_type', 'attendee_fullname', 'attendee_email', 'attendee_phone']
+
+    attendee_details_headers = []
+    for item in item_collection.items:
+        if item.assignee_details:
+            for detail in item.assignee_details.keys():
+                if detail not in attendee_details_headers:
+                    attendee_details_headers.append(detail)
+
+    headers, rows = item_collection.fetch_assignee_details
+    headers.extend(attendee_details_headers)
+
+    def row_handler(row):
+        # Convert row to a dict
+        dict_row = {}
+        for idx, value in enumerate(row):
+            # Value is a dict already, so just copy
+            if isinstance(value, dict):
+                dict_row.update(value)
+            # Value is a string, add it to the dict with the corresponding key
+            else:
+                dict_row[headers[idx]] = value
+        # Localize datetime
+        if dict_row.get('date'):
+            dict_row['date'] = format_datetime(localize_timezone(dict_row['date']), format='long', locale=get_locale())
+        return dict_row
+
+    csv_headers = list(headers)
+    # Remove 'attendee_details' from header
+    if 'attendee_details' in headers:
+        del csv_headers[csv_headers.index('attendee_details')]
+
+    return csv_response(csv_headers, rows, row_type='dict', row_handler=row_handler)
```

Comment:
You've removed `attendee_details` from csv_headers, but _not_ from `rows`. The data could be inconsistent.